release-notes/6.0/api-diff/.Net/6.0.0_System.Drawing.md

```diff
@@ -0,0 +1,31 @@
+# System.Drawing
+
+``` diff
+ namespace System.Drawing {
+     public readonly struct Color : IEquatable<Color> {
++        public static Color RebeccaPurple { get; }
```

Comment:
It's funny the APIs we add.

Reply:
How does one sign up to get a color named after themselves?

Reply:
https://medium.com/@valgaze/the-hidden-purple-memorial-in-your-web-browser-7d84813bb416

Reply:
Ok, so, less funny, and not something to sign up for 😦